tests/workflow/test_construct_batch.py

```diff
@@ -140,7 +140,7 @@ def circuit(x):
             return qml.expval(qml.PauliZ(0))
 
         full_prog = get_transform_program(circuit)
-        assert len(full_prog) == 13
+        assert len(full_prog) == 12
```

Comment:
What made the transform program smaller?

Reply:
Previously there was a `validate_observables` transform, now that is removed and `validate_measurements` now validates the observables as well depending on what the measurement is